### DISCUSSION TITLE
version >= 8.0 , agent.instance_uuid was caneled

### DISCUSSION THREAD

**Discussion by dongyinuo** [apache/skywalking#8762] [Q&A] [answered]
why agent.instance_uuid was caneled, when skywalking > 8.0 ?

when I deploy multiple applications, I need to formulate instance_name ?

**Comment by wu-sheng** [comment, marked as answer]
UUID is never hard required, it is optional always. `instance_name` is provided if you want to custom name, otherwise, UUID will be generated automatically.
